src/components/admin/HearingToolbar.js

```diff
@@ -38,7 +36,7 @@ class HearingToolbar extends React.Component {
     ];
     const open = isOpen(hearing);
     if (open && hearing.published) {
-      statusLabel = <FormattedMessage id="published"/>;
+      statusLabel = (<Alert bsStyle="success"><Icon name="eye"/> <FormattedMessage id="published"/></Alert>);
```

Comment:
Don't know if the backgrounds are overkill, but I think the green background is too dark on my display. The black text "Julkaistu" doesn't stand out as it should. The lighter backgrounds work alright.